pkg/importer/importer_test.go

```diff
@@ -88,6 +94,14 @@ var _ = Describe("Importer", func() {
 		})
 
 		tests := []testT{
+			{
+				descr:       "use file",
+				filename:    "test-local",
+				createFile:  false,
+				expected:    "file dataStream",
+				endpoint:    "file:///tmp/importer-test/",
```

Comment:
why use a directory path here vs. a file? I know it's mock but maybe something like `file:///tmp/fake-file` is clearer? I got confused by _/tmp/importer-test/_, which is a `const`, is the destination path for the copy.

Reply:
yep, agreed, just an oversight, changed in latest commit